styles/base/base.mcss

```diff
@@ -42,9 +45,13 @@ a {
 * {
   box-sizing: border-box
 }
-input, textarea, keygen, select, button {
-  font-family: '.SFNSText-Regular', sans-serif
```

Comment:
was this intentional? https://github.com/ssbc/patchwork/pull/1094/files#r294034847